application/src/main/webapp/vue-app/components/ExoChatMessageComposer.vue

```diff
@@ -347,7 +348,27 @@ export default {
       chatServices.sendMentionNotification(this.contact.room, this.contact.fullName, this.mentionedUsers);
       this.mentionedUsers = [];
       return message;
-    }
+    },
+    paste(e) {
+      // consider the first item (can be easily extended for multiple items)
+      const item = e.clipboardData.items[0];
+      //test if the type of item e
+      if (item.type.indexOf('image') === 0) {
+        const pastedImage = item.getAsFile();
+        const reader = new FileReader();
+        reader.onload = function (event) {
+          document.getElementById('container').src = event.target.result;
```

Comment:
What is this element **container** created ? is it a hidden element ? I think you need to create one that put the image inside ?